web/ui/react-app/src/pages/targets/ScrapePoolPanel.tsx

```diff
@@ -24,26 +37,16 @@ const ScrapePoolPanel: FC<PanelProps> = ({ scrapePool, targetGroup }) => {
     href: `#${id}`,
     id,
   };
-  const btnProps = {
-    children: `show ${expanded ? 'less' : 'more'}`,
-    color: 'primary',
-    onClick: (): void => setOptions({ expanded: !expanded }),
-    size: 'xs',
-    style: {
-      padding: '0.3em 0.3em 0.25em 0.3em',
-      fontSize: '0.375em',
-      marginLeft: '1em',
-      verticalAlign: 'baseline',
-    },
-  };
+  ToggleButtonProps.onClick = (): void => setOptions({ expanded: !expanded });
+  ToggleButtonProps.children = `show ${expanded ? 'less' : 'more'}`;
 
   return (
     <div className={styles.container}>
       <h3>
```

Comment:
Maybe the whole h3 container can be a component used from Service discovery page and Targets page with possibility to pass title as children - <a> for targets and <strong> for SD.

Reply:
sure

Reply:
For now I would prioritize getting this PR done and merged, since it is taking quite a while already, but happy to have more refactorings later.